soccer/gameplay/robocup-py.cpp

```diff
@@ -459,6 +459,11 @@ void WinEval_add_excluded_robot(WindowEvaluator* self, Robot* robot) {
 
 BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(Point_overloads, normalized, 0, 1)
 
+float Point_get_x(const Geometry2d::Point* self) { return self->x(); }
+float Point_get_y(const Geometry2d::Point* self) { return self->y(); }
```

Comment:
I agree that having to wrap functions like this is pretty crappy, but I don't think your alternative syntax works.  It works fine for the getter, but the setter still needs to be a wrapper function.  Are you able to get something like this to work?